go/integration/hcvault/hcvault_aead_internal_test.go

```diff
@@ -17,75 +17,102 @@
 package hcvault
 
 import (
+	"fmt"
 	"testing"
 )
 
-func TestExtractKey(t *testing.T) {
+func TestGetPaths(t *testing.T) {
```

Comment:
`TestGetEndpointPaths`

Reply:
D'oh, remnant from previous iteration. Fixed in 65ddde686.